weaviate_agents/query/query_agent.py

```diff
@@ -52,7 +52,7 @@ def __init__(
     def run(
         self,
         query: str,
-        view_properties: Optional[List[str]] = None,
+        view_properties: Optional[dict[str, list[str]]] = None,
```

Comment:
Since this needs to be compatible with the `weaviate-client` we have to adhere to the older `from typing ...` typing structure so can we make `Optional[dict[str, list[str]]]` -> `Optional[Dict[str, List[str]]]`